pkg/pb/inspector.proto

```diff
@@ -198,3 +206,33 @@ message DashboardResponse {
   google.protobuf.Timestamp last_pinged = 8;
   google.protobuf.Timestamp last_queried = 9;
 }
+
+message SegmentHealthRequest {
+  bytes bucket = 1;         // segment bucket
+  bytes encrypted_path = 2; // segment encrypted path
+  int64 segment_index = 3;  // segment index
+  bytes project_id = 4;     // segment project id
+}
+
+message SegmentHealthResponse{
+  int32 minimum_required = 1;                 // minimum required for reconstruction (k)
+  int32 total = 2;                            // total amount of pieces we generated (n)
+  int32 repair_threshold = 3;                 // amount of pieces we need to drop to before triggering repair (m)
+  int32 success_threshold = 4;                // amount of pieces we need to store to call it a success (o)
+  int32 online_nodes = 5;                     // amount of pieces with nodes that are online
+  string segment = 6;                         // path formatted segment index
```

Comment:
`bytes` all paths should be `bytes`. I know there's little danger here, but for consistency.